sklearn/utils/tests/test_multiclass.py

```diff
@@ -31,18 +30,24 @@
     unique_labels,
 )
 
-sparse_multilable_explicit_zero = csc_matrix(np.array([[0, 1], [1, 0]]))
-sparse_multilable_explicit_zero[:, 0] = 0
+multilabel_explicit_zero = np.array([[0, 1], [1, 0]])
+multilabel_explicit_zero[:, 0] = 0
 
 
 def _generate_sparse(
-    matrix,
-    matrix_types=(csr_matrix, csc_matrix, coo_matrix, dok_matrix, lil_matrix),
+    data,
+    sparse_containers=[
+        *COO_CONTAINERS,
+        *CSC_CONTAINERS,
+        *CSR_CONTAINERS,
+        *DOK_CONTAINERS,
+        *LIL_CONTAINERS,
+    ],
```

Comment:
```suggestion
    sparse_containers=
        COO_CONTAINERS,
        + CSC_CONTAINERS
        + CSR_CONTAINERS
        + DOK_CONTAINERS
        + LIL_CONTAINERS,
```

Reply:
It needs to be a tuple since this is a default parameter of a function and it should be a mutable.

Reply:
Ah okay, I think I've made this mistake in many other PRs. I will update them as well ASAP.

BTW do you mean that "it should be an immutable"?